modules/after-the-deadline/rtl/atd-rtl.css

```diff
@@ -23,8 +23,8 @@
 /* Menu styles derived from:
  * jquery.spellchecker.js - a simple jQuery Spell Checker
  * Copyright (c) 2008 Richard Willis
- * MIT license  : http://www.opensource.org/licenses/mit-license.php
- * Project      : http://jquery-spellchecker.googlecode.com
+ * MIT license  : https://www.opensource.org/licenses/mit-license.php
+ * Project      : https://code.google.com/archive/p/jquery-spellchecker/
```

Comment:
While we're changing it, it should be https://github.com/badsyntax/jquery-spellchecker now